PWGCF/EbyEFluctuations/Tasks/RobustFluctuationObservables.cxx

```diff
@@ -702,32 +744,52 @@ struct RobustFluctuationObservables {
         int ROF_BC_center = nITSROF_BC_offset + iROF * nBCsPerOrbit / nITSROF;
         if (fabs(static_cast<int>(collBC) - ROF_BC_center) <= nITSROF_BC_cutWidth / 2) {
           flagBCisNotInHandmadeBoundariesITSROF = false;
-          if (TFid < 2)
-            cout << "QA: collBC = " << collBC << ", nITSROF_BC_offset = " << nITSROF_BC_offset
-                 << ", ROF_BC_center = " << ROF_BC_center << ", nITSROF_BC_cutWidth = " << nITSROF_BC_cutWidth
-                 << ", fabs(collBC - ROF_BC_center) = " << fabs(static_cast<int>(collBC) - ROF_BC_center)
-                 << ", (int)collBC - ROF_BC_center = " << static_cast<int>(collBC) - ROF_BC_center
-                 << endl;
+          // if (TFid < 2)
+          //   LOGF(info, "QA: collBC = %d, nITSROF_BC_offset = %d, ROF_BC_center = %d, nITSROF_BC_cutWidth = %d, fabs(collBC - ROF_BC_center) = %d, (int)collBC - ROF_BC_center = %d",
+          //        collBC, nITSROF_BC_offset, ROF_BC_center,
+          //        nITSROF_BC_cutWidth, fabs(static_cast<int>(collBC) - ROF_BC_center),
+          //        static_cast<int>(collBC) - ROF_BC_center);
```

Comment:
Please, don't forget to remove commented code
It decreases the readability of the code in case someone wanted to learn from it